AgentQnA/tests/_test_agentqna_on_xeon.sh

```diff
@@ -3,73 +3,48 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
`_test_agentqna_on_xeon.sh` --> `test_agentqna_on_xeon.sh`

Reply:
@chensuyue This _on_xeon test is for testing will OpenAI GPT-4 models. We discussed about this before - you said you will ask for OpenAI resources and after that we will activate this test.